### DISCUSSION TITLE
Not returning ID with Title

### DISCUSSION THREAD

**Discussion by rahepler** [ucfopen/canvasapi#577] [Q&A]
Anyone know if there is a way to not return the ID appended to the end of assignment or module titles? 

Currently, it returns 'This is the TITLE - Title  (1234567)'

I would only like the title.

**Comment by Thetwam** [comment]
Hey @rahepler,

Try getting the `name` attribute from an assignment or module, e.g. `assignment.name`.

The default value returned when printing an object (or otherwise making it a string) is just for developer reference and is not meant for normal use.
